source/win-main/App.vue

```diff
@@ -766,6 +804,9 @@ export default defineComponent({
             this.$closePopover()
           }
         })
+    },
+    customizeToolbarVisible: function (configName: string): boolean {
```

Comment:
Is this REALLY reactive? (I still can't believe it, but I do admit I did not yet test it out myself 🙈)

Reply:
Yes, it worked well.
At first I tried to find out how to input variables into computed. The answer I got was that while I could do it with a closure method, e.g.

```
computed: {
	closure() {
		return function (a, b, c) {
			/** do something */
			return data
		}
	}
}
```

But based on the principle of Vue
https://vuejs.org/guide/essentials/computed.html
This is no different from writing directly in the method. So I just did it。

But this seems to give up the cached data, maybe it will cause the program to be slow? I didn't find this to be the case during testing. Do you have a good way to test.

Reply:
By the way, the method name `customizeToolbarVisible` is not good, too. As above, how about `getToolbarButtonDisplay`